espresso-server/app/build.gradle

```diff
@@ -2,10 +2,16 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 android {
+    def props = getProperties()
     compileSdkVersion getIntegerProperty('appiumCompileSdk', 28)
     buildToolsVersion getStringProperty('appiumBuildTools', '28.0.3')
     defaultConfig {
-        applicationId "io.appium.espressoserver"
+        // <instrumentation android:targetPackage=""/>
```

Comment:
is the runner name not needed?

Reply:
We are specifying `testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"` below. Or is it another runner?

Reply:
ah, then it's fine